sktime/forecasting/nnetsaucemts.py

```diff
@@ -0,0 +1,738 @@
+    ):
+        self.obj = self.regressor = regressor
+        self.n_hidden_features = n_hidden_features
+        self.activation_name = activation_name
+        self.a = a
+        self.nodes_sim = nodes_sim
+        self.bias = bias
+        self.dropout = dropout
+        self.direct_link = direct_link
+        self.n_clusters = n_clusters
+        self.cluster_encode = cluster_encode
+        self.type_clust = type_clust
+        self.type_scaling = type_scaling
+        self.lags = lags
+        self.type_pi = type_pi
+        self.block_size = block_size
+        self.replications = replications
+        self.kernel = kernel
+        self.agg = agg
+        self.seed = seed
+        self.backend = backend
+        self.verbose = verbose
+        self.show_progress = show_progress
+
+        # leave this as is
+        super().__init__()
+
+        from nnetsauce import MTS
+
+        self.fitter = MTS(
+            obj=self.obj,
+            n_hidden_features=self.n_hidden_features,
+            activation_name=self.activation_name,
+            a=self.a,
+            nodes_sim=self.nodes_sim,
+            bias=self.bias,
+            dropout=self.dropout,
+            direct_link=self.direct_link,
+            n_clusters=self.n_clusters,
+            cluster_encode=self.cluster_encode,
+            type_clust=self.type_clust,
+            type_scaling=self.type_scaling,
+            lags=self.lags,
+            type_pi=self.type_pi,
+            block_size=self.block_size,
+            replications=self.replications,
+            kernel=self.kernel,
+            agg=self.agg,
+            seed=self.seed,
+            backend=self.backend,
+            verbose=self.verbose,
+            show_progress=self.show_progress,
+        )
+        # IMPORTANT: the self.params should never be overwritten or mutated from now on
+        # for handling defaults etc, write to other attributes, e.g., self._parama
+
+        # todo: optional, parameter checking logic (if applicable) should happen here
+        # if writes derived values to self, should *not* overwrite self.parama etc
+        # instead, write to self._parama, self._newparam (starting with _)
+
+    # todo: implement this, mandatory
+    def _fit(self, y, X=None, fh=None):
+        """Fit forecaster to training data.
+
+        private _fit containing the core logic, called from fit
+
+        Writes to self:
+            Sets fitted model attributes ending in "_".
+
+        Parameters
+        ----------
+        y : pd.DataFrame
+            if self.get_tag("scitype:y")=="univariate":
+                guaranteed to have a single column
+            if self.get_tag("scitype:y")=="both": no restrictions apply
+        X : pd.DataFrame, optional (default=None)
+            Exogeneous time series to fit to.
+
+        Returns
+        -------
+        self : reference to self
+        """
+        self.fitter.fit(y)
+        self.obj = self.fitter.obj
+        return self
+
```

Comment:
Please correct me if I am missing something, but you are not using `X`, but above you have marked `ignores-exogeneous-X` tag as `False`. Why should it not be `True`?